plugins/modules/storage/zfs/zpool_facts.py

```diff
@@ -153,7 +146,7 @@ def get_facts(self):
         if self.name:
             cmd.append(self.name)
 
-        (rc, out, err) = self.module.run_command(cmd)
+        rc, out, err = self.module.run_command(cmd, check_rc=True)
 
         if rc == 0:
```

Comment:
This should be removed, and the folowing lines dedented.

Reply:
Done